lib/challenge.js

```diff
@@ -19,7 +19,15 @@
   // - park
 
 // Answer Starts Here
-
+'Calendar'
+Description: our app uses a calendar to keep track of appointments.
+Every 'calendar' has
+- an appointment name
+- an appointment date
+- an appointment location  (ok, that is three, I tend to use calendars a bit).
+Every 'calendar' can
+- set busy/free status
+- invite another person to the appointment
```

Comment:
This abstraction isn't bad, especially if you make it an "appointment" instead of a "calendar", but it's only one. Can you think of any others?